Documentation/tectonic.md

```diff
@@ -0,0 +1,52 @@
+# Rook on Tectonic
+
+Here is a running guide on how to implement Rook on Tectonic. A complete guide on how to install Tectonic is out of the scope of the Rook project. But many info can be found on the [Tectonic website](https://coreos.com/tectonic/docs/latest/)
+
+## Prequisites
+
+- An installed tectonic-installer. These steps are described on [the Tectonic website](https://coreos.com/tectonic/docs/latest/install/bare-metal/#4-tectonic-installer)
+- A running matchbox node which will do the provisioning (Bare metal)
+- You can run through all steps of the GUI installer, but in the last step, choose `Boot manually`. This way we can make the necessary changes first.
+
+## Edit the kubelet.service file to be compatible with Rook
+
+To make the kubelet compatible with Rook we need to make a few adaptions to the systemd service file generated by the Tectonic-installer.
+
+First cd into the directory where you untarred the tectonic installer and find your newly generated cluster configuration files.
+
+```
+ $ cd ~/tectonic/tectonic-installer/LINUX-OR-DARWIN/clusters
+ $ nano modules/ignition/resources/services/kubelet.service
+```
+
+And after the line `ExecStartPre=/bin/mkdir -p /var/lib/cni`, paste the following extra lines:
+
+```
+ExecStartPre=/bin/mkdir -p /var/lib/kubelet/volumeplugins
+ExecStartPre=/bin/mkdir -p /var/lib/rook
+```
+
+And a little down in the `ExecStart` section, add the following kubelet option:
+
+```
+--volume-plugin-dir=/var/lib/kubelet/volumeplugins \
+```
+
+Save the file and close.
+
+### Provision your matchbox
+
+Still from this cluster directory, invoke the following commands:
```

Comment:
I don't think adding those tectonic-installer specific commands here is a good idea when those commands may change in the future.
Linking back to the specific steps in the Tectonic tutorial/guide from CoreOS is enough.